sia/upload_test.go

```diff
@@ -39,4 +56,10 @@ func testUploadFile(t *testing.T, c *Core) {
 	if c.hostDB.Size() < 1 {
 		t.Fatal("Hostdb got pruned while trying to make a contract?")
 	}
+
+	// Check that the file can be downloaded.
+	err = c.renter.Download("one", "renterDownload")
+	if err != nil {
+		t.Error(err)
+	}
```

Comment:
`os.Remove()`? I don't like having stuff like this cluttering the gitignore. If you really need to inspect it, commenting out the `Remove` isn't a big deal.